media/js/firefox/whatsnew/whatsnew-88-en.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var jingle = function(){
+        button.style.fontStyle = 'italic';
+        console.log('jingle');
+        var audio = new Audio(button.dataset.audio);
+        audio.play();
+    };
+
+    var button = document.getElementById('outatime');
+
+    button.addEventListener('click', jingle);
+
+    var href = window.location.href;
+
+    var initTrafficCop = function () {
```

Comment:
We should put all of the experiment code into a separate JS file and bundle, and put that bundle behind a switch to we can quickly disable the experiment if we need to.

Reply:
Moved behind the switch  `whatsnew-88-experiment`. Do I need to add this switch somewhere specific?

Reply:
In this case the only thing the switch controls is whether the experiment JS bundle loads or not, so you've got the conditional in the right place.

Switches are managed in https://github.com/mozmeao/www-config/pull/352. Docs at https://www-config.readthedocs.io/ and here's a recent PR for flipping a switch: https://github.com/mozmeao/www-config/pull/352